modules/administration-guide/partials/proc_configuring-a-che-workspace-with-a-persistent-volume-strategy.adoc

```diff
@@ -116,10 +116,10 @@ $ {orch-cli} apply -f _<my-cluster.yaml>_
 [subs="+quotes,+attributes"]
 ----
 $ {orch-cli} patch checluster {prod-checluster} --type=json \
-  -p '[{"op": "replace", "path": "/spec/storage/pvcStrategy", "value": "__<per-workspace>__"}]'
+  -p '[{"op": "replace", "path": "/spec/storage/pvcStrategy", "value": "__per-workspace__"}]'
 ----
 
-Depending on the strategy used, replace the `_<per-workspace>_` option in the above example with `unique` or `common`.
+Depending on the strategy used, replace the `_per-workspace_` option in the above example with `unique` or `common`.
```

Comment:
I guess we can also use value instead of option?
```suggestion
Depending on the strategy used, replace the `per-workspace` value in the above example with `unique` or `common`.
```